infra/notifications/src/mobilePush.js

```diff
@@ -47,21 +59,95 @@ if (process.env.FIREBASE_SERVICE_JSON) {
 }
 
 //
+// Mobile Push notifications for Messages
+//
+async function messageMobilePush(receivers, sender, config) {
+  if (!receivers) throw new Error('receivers not defined')
+  if (!sender) throw new Error('sender not defined')
+
+  // TODO: Move to config
+  const payload = {
+    url: config.dappMessagesUrl
+  }
+
+  // Get ID of sender
+  const messageSender = Identity.findOne({
```

Comment:
These need an `await` in front of them because this should return a promise

Reply:
Added those!

Reply:
Thanks!

Reply:
Sorry @wanderingstan I totally misread the code here, I see they don't need this because of the Promise.all, removed!